RecipesBook/RecipesBook/RecipesBook.Android/SlpashScreen.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using MvvmCross.Forms.Platforms.Android.Views;
+using RecipesBook.UI;
+using Android.Content.PM;
+using System.Threading.Tasks;
+
+namespace RecipesBook.Droid
+{
+    [Activity(
+        Label = "Recipes Book", 
+        Theme = "@style/MainTheme.Splash", 
+        MainLauncher = true,
+        ScreenOrientation = ScreenOrientation.Portrait,
```

Comment:
You can add NoHistory = true to avoid backstack navigation to splash screen.

Reply:
Done